.circleci/config.yml

```diff
@@ -19,6 +19,18 @@ commands:
           event: fail
           template: basic_fail_1
           branch_pattern: main
+  notify-failures-on-validate-all:
+    description: "Notify Slack on any chain validation failures"
+    parameters:
+      channel:
+        type: string
+        default: C03N11M0BBN # TODO - modify this to a new slack channel
```

Comment:
TODO - **change the slack channel id once a new one is created**

Reply:
(Optional suggestion) I wonder if it is worth factoring this out into a reusable step, taking the slack channel ID as a parameter (since that is the only difference between this step and the `notify-failues-on-main` step.